.circleci/config.yml

```diff
@@ -64,8 +64,11 @@ jobs:
           name: Run Unit test
           command: ./gradlew --no-daemon test
       - run:
-          name: Compile Instrumentation test
-          command: ./gradlew --no-daemon assembleAndroidTest
+          name: Generate reports
+          command: ./gradlew jacocoTestReport
+      - run:
+          name: Send coverage
+          command: bash <(curl -s https://codecov.io/bash)
```

Comment:
Did you have to setup an account with codecov to get this to work?

Reply:
Yes, @ear-dev I have setup a codecov account. [Here](https://codecov.io/gh/GOVINDDIXIT/Rocket.Chat.Android/commits ) is the link for the same for reference.